omod-1.8/src/main/java/org/openmrs/module/webservices/rest/web/v1_0/search/openmrs1_8/ConceptSearchHandler1_8.java

```diff
@@ -73,9 +75,18 @@ public PageableResult search(RequestContext context) throws ResponseException {
 		String name = context.getParameter("name");
 		String conceptClass = context.getParameter("class");
 		String searchType = context.getParameter("searchType");
-		
+		String uuid = context.getParameter("uuid");
+		String reference = context.getParameter("reference");
+
 		List<Concept> concepts = new ArrayList<Concept>();
 		
+		ConceptSource cptSource = conceptService.getConceptByReference(reference);
```

Comment:
@dkayiwa am getting an error when getting concept by reference and the log shows the method doesn't exit in core. Yet the method is [exists](https://github.com/openmrs/openmrs-core/blob/ffa3ebe927c18e29ec3c31627e050b99691b25fe/api/src/main/java/org/openmrs/api/impl/ConceptServiceImpl.java#L345).

Is there anywhere am missing out? Kindly help

Reply:
Did you understand this comment? https://github.com/openmrs/openmrs-module-webservices.rest/pull/556/#issuecomment-1162168774

Reply:
> Did you understand this comment? #556

The link is taking me to the actual PR. Which comment is are you specifically referring to?

Reply:
Are you probably referring to this [comment](https://github.com/openmrs/openmrs-module-webservices.rest/pull/556#issuecomment-1162893119)?